new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java

```diff
@@ -185,8 +204,21 @@ public void beforeAll(ExtensionContext context) {
           images.add(WLS_BASE_IMAGE_NAME + ":" + WLS_BASE_IMAGE_TAG);
           images.add(JRF_BASE_IMAGE_NAME + ":" + JRF_BASE_IMAGE_TAG);
           images.add(DB_IMAGE_NAME + ":" + DB_IMAGE_TAG);
+          /*
```

Comment:
Don't we still need the "PushToKind" side of this functionality?

Reply:
I think I've misread this -- you didn't comment out that code, instead, you've added the entire comment... why?

Reply:
I think I need more than what I changed here. This PR is not ready for merge yet. I created a PR so you could get the changes to the 2 files I mentioned. I will have to still fix ImageBuilders.java. Running it locally once before pushing the changes.